storage/statedb/cache.go

```diff
@@ -0,0 +1,11 @@
+package statedb
+
+import "github.com/VictoriaMetrics/fastcache"
+
+// Cache interface the cache of stateDB
+type Cache interface {
+	Set(k, v []byte)
+	Get(dst, k []byte) []byte
+	HasGet(dst, k []byte) ([]byte, bool)
+	UpdateStats(s *fastcache.Stats)
```

Comment:
These interface methods can be changed into more general ones like `Get(key []byte)`, `Has(key []byte)` ... 
Also, `UpdateStats` depends on the implementation of `fastcache`.

Reply:
Generalized them.